app/templates/editor/level/modal/load-branch-modal.jade

```diff
@@ -0,0 +1,50 @@
+extends /templates/core/modal-base-flat
+
+block modal-header-content
+  h3 Load Branch
+
+block modal-body-content
+  .container-fluid
+    .row
+      .col-sm-2
+        #branches-list-group.list-group
+          for branch, i in view.branches.models
+            a.list-group-item(class=i == 0 ? 'active' : '', data-branch-cid=branch.cid)
```

Comment:
I prefer to put parens around this kinda ternary expression, makes it easier to parse here